sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpTransactionGroup.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.Azure.Amqp;
+using Microsoft.Azure.Amqp.Transaction;
+
+namespace Azure.Messaging.ServiceBus.Amqp
+{
+    internal class AmqpTransactionGroup
+    {
+        public FaultTolerantAmqpObject<AmqpSession> Session { get; }
+
+        public FaultTolerantAmqpObject<Controller> Controller { get; }
+
+        public AmqpTransactionGroup(FaultTolerantAmqpObject<AmqpSession> session, FaultTolerantAmqpObject<Controller> controller)
```

Comment:
nit:  Did you mean to wrap the parameters?

Reply:
I think you are enjoying that a little too much 😆

Reply:
and actually no.. I only wrap when it takes up a lot of horizontal space - usually 3 or more params.